packages/api/src/controllers/stripe.ts

```diff
@@ -41,7 +41,18 @@ export const reportUsage = async (req, adminToken) => {
     }
   );
 
-  const payAsYouGoUsers = [...users, ...oldProPlanUsers];
+  // Current date in milliseconds
+  let currentDateMillis = new Date().getTime();
+  let oneMonthMillis = 31 * 24 * 60 * 60 * 1000;
+
+  // One month ago unix millis timestamp
+  let cutOffDate = currentDateMillis - oneMonthMillis;
```

Comment:
**nit:** Use `const` instead of `let` whenever you won't need to update the variables. Makes the code generally safer and easier to read on colorful IDEs (consts are usually displayed differently than mutable values)